src/schema/v2/partner/partner.ts

```diff
@@ -222,6 +228,54 @@ export const PartnerType = new GraphQLObjectType<any, ResolverContext>({
           })
         },
       },
+      partnerAlertsConnection: {
+        type: PartnerAlertsConnectionType,
+        args: pageable({
+          page: {
+            type: GraphQLInt,
+          },
+          size: {
+            type: GraphQLInt,
+          },
+          totalCount: {
+            type: GraphQLBoolean,
+          },
```

Comment:
**thought:** It strikes me as un-idiomatic for an MP field to take `totalCount` as an arg. I see [only one case](https://github.com/artsy/metaphysics/blob/d633421c4969f024f45d2c38e131f0b28f9801fa/src/schema/v2/fair.ts#L259-L262) of this but many, many more where we request the total count from Gravity by default (search repo for `total_count: true`). 

I think some pagination logic even relies on that total count being present, so perhaps better to include it by default?

Furthermore… I believe this query is ultimately fulfilled by ES, yes? If so I think that total count is _always_ computed and doesn't even make sense as an arg (in which case [this arg](https://github.com/artsy/gravity/blob/a0e2a494b58e911c5a964337d0534a80491c14d6/app/api/v1/partner_search_criteria_endpoint.rb) over in the upstream Gravity endpoint might be totally superfluous).

Reply:
Good points! 
Will do some digging and clean up

Reply:
Should be addressed 🙏🏻